js/menu.js

```diff
@@ -0,0 +1,22 @@
+// Hamburger menu
+const btn = document.querySelector('#hamburger');
+const nav = document.querySelector('.nav nav');
+const openNav = document.querySelector('.menu-display');
+const closeBtn = document.querySelector('.xicon');
+const menuOpt = document.querySelectorAll('#items-menus li a');
+
+const toggleMenuDisplay = () => {
+  if (nav.style.display !== 'none') {
+    nav.style.display = 'none';
+    openNav.style.display = 'flex';
+  } else {
+    nav.style.display = 'grid';
+    openNav.style.display = 'none';
+  }
+};
+
+btn.addEventListener('click', toggleMenuDisplay);
+closeBtn.addEventListener('click', toggleMenuDisplay);
+menuOpt.forEach((link) => {
+  link.addEventListener('click', toggleMenuDisplay);
+});
```

Comment:
- [ ] Good to add some javascript functional :+1: But please I have noticed that after the page has loaded, your mobile menu displays which shouldn't be that because one must first see the hamburger icon after the page load before he/she can click to show the mobile menu. So make sure that you fix this please :pray: